### DISCUSSION TITLE
how do i acces data

### DISCUSSION THREAD

**Discussion by MLGBEESTMODE** [HypixelDev/PublicAPI#397] [Q&A]
i used the link https://api.hypixel.net/skyblock/bazaar/products?key=(key)&productId=ENCHANTED_GOLD_BLOCK
and it just gives me item id's, i tried to look the answer and the person helping gave this link https://github.com/HypixelDev/PublicAPI/blob/master/Documentation/methods/skyblock/bazaar.md
but it doesent tell me what to do, to get that data. can someone please help?

**Comment by AzureAaron** [comment]
You can access information from the Skyblock Bazaar at: **https://api.hypixel.net/skyblock/bazaar** (no key parameter is needed for this endpoint)

The documentation for the endpoint is [here](https://github.com/HypixelDev/PublicAPI/blob/db26b5fd3b7bb29da14e40e6d211143ec44a4519/Documentation/methods/skyblock/bazaar.md).

The URL that you are trying to access bazaar data is used for listing all the Skyblock IDs of the products on the bazaar and has nothing to do with bazaar information- not to mention it is also deprecated and will be removed in the future.
